src/backend/base/langflow/inputs/input_mixin.py

```diff
@@ -72,6 +72,9 @@ class BaseInputMixin(BaseModel, validate_assignment=True):  # type: ignore[call-
     dynamic: bool = False
     """Specifies if the field is dynamic. Defaults to False."""
 
+    dialog_inputs: list[Any] | None = None
+    """List of dialog inputs for the field. Default is an empty list."""
+
```

Comment:
I just saw that this is in BaseInputMixin. If it only works in str and dropdown, shouldn't be in another mixin?